src/scipp/signal/__init__.py

```diff
@@ -56,9 +56,11 @@ def butter(da: DataArray, dim: str, *, N, Wn, **kwargs):
     """
     coord = da.coords[dim]
     fs = _frequency(coord).value
-    if not Wn.unit == one / coord.unit:
+    try:
+        Wn = to_unit(Wn, one / coord.unit)
```

Comment:
`Wn = to_unit(Wn, one / coord.unit, copy=False)`?

Reply:
Irrelevant for such tiny arrays?

Reply:
True